.reek

```diff
@@ -43,6 +43,7 @@ TooManyStatements:
   max_statements: 6
   exclude:
     - Users::PhoneConfirmationController
+    - OpenidConnect::AuthorizationController
```

Comment:
ok I have added the class length + method count exclusions in lieu of the method length error.